metacat-functional-tests/src/functionalTestShaded/groovy/com/netflix/metacat/MetacatSmokeSpec.groovy

```diff
@@ -0,0 +1,1467 @@
+/*
+ *  Copyright 2019 Netflix, Inc.
+ *
+ *     Licensed under the Apache License, Version 2.0 (the "License");
+ *     you may not use this file except in compliance with the License.
+ *     You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *     Unless required by applicable law or agreed to in writing, software
+ *     distributed under the License is distributed on an "AS IS" BASIS,
+ *     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *     See the License for the specific language governing permissions and
+ *     limitations under the License.
+ */
+package com.netflix.metacat
+
+import com.netflix.metacat.client.Client
+import com.netflix.metacat.client.api.MetacatV1
+import com.netflix.metacat.client.api.MetadataV1
+import com.netflix.metacat.client.api.PartitionV1
+import com.netflix.metacat.client.api.TagV1
+import com.netflix.metacat.common.QualifiedName
+import com.netflix.metacat.common.dto.*
+import com.netflix.metacat.common.exception.MetacatAlreadyExistsException
+import com.netflix.metacat.common.exception.MetacatBadRequestException
+import com.netflix.metacat.common.exception.MetacatNotFoundException
+import com.netflix.metacat.common.exception.MetacatNotSupportedException
+import com.netflix.metacat.common.exception.MetacatPreconditionFailedException
+import com.netflix.metacat.common.exception.MetacatUnAuthorizedException
+import com.netflix.metacat.common.json.MetacatJson
+import com.netflix.metacat.common.json.MetacatJsonLocator
+import com.netflix.metacat.common.server.connectors.exception.InvalidMetaException
+import com.netflix.metacat.connector.hive.util.PartitionUtil
+import com.netflix.metacat.testdata.provider.PigDataDtoProvider
+import com.netflix.metacat.shaded.feign.Request
+import com.netflix.metacat.shaded.feign.Retryer
+import groovy.sql.Sql
+import org.apache.commons.io.FileUtils
+import org.joda.time.Instant
+import spock.lang.Ignore
+import spock.lang.Shared
+import spock.lang.Specification
+import spock.lang.Unroll
+
+import java.util.concurrent.TimeUnit
+
+/**
+ * MetacatSmokeSpec.
+ * @author amajumdar
+ * @author zhenl
+ * @since 1.0.0
+ */
+class MetacatSmokeSpec extends Specification {
+    public static MetacatV1 api
```

Comment:
Are these tests are the same as the existing functional test except for testing shaded jar?

Reply:
Right.